.github/workflows/update_version.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install wheel
+          pip install scikit-build-core
```

Comment:
```suggestion
```

Reply:
`scikit-build-core` and other build-time dependencies will be pulled in with the installation step below this line.